gradle-plugin/build.gradle

```diff
@@ -33,3 +34,12 @@ afterEvaluate {
     }
   }
 }
+
+// publish to local maven to test sample projects
+publishing {
```

Comment:
Have you tried `./gradlew install`? We use this command to install an artifact to Maven local.